include/eve/arch/cpu/wide.hpp

```diff
@@ -931,81 +931,93 @@ namespace eve
     }
 
     //! @brief Element-wise less-than comparison between eve::wide
-    friend EVE_FORCEINLINE logical<wide> operator<(wide v, wide w) noexcept
+    friend EVE_FORCEINLINE auto operator<(wide v, wide w) noexcept
+    requires( !kumi::product_type<Type> )
```

Comment:
I thought we decided that you need to opt out?

Reply:
Simplified later on